### ISSUE TITLE
Missing multi-stream download controls after upgrading

### ISSUE THREAD

**Issue by dca00** [downthemall/downthemall#359]
Whatever happened to multi-stream downloads? Their controls are gone after upgrading.
I tried adding screenshots, but since this site is now Microsoft, it is perfectly normal that it does not work, and they throw their signature error "Something went really wrong, and we can’t process that file."
So you would have to launch your old and new versions side by side, start a download, and see the difference: in the old version there are segment add and segment remove buttons, in the new version they do not exist.

**Comment by dca00** [comment]
Experimenting more, I am no longer seeing any advantage of using DownThemAll for downloading. It is clearly not using multi-stream. Downloads with DTA show the same speed as regular browser downloads, whereas previously the same resource doubled, tripled, and quadrupled d/l speed, in linear progression to the number of streams.
Also I am observing that DTA simply adds a d/l to the standard browser's d/l window.
Bottom line, either DTA is now badly broken, or multi-stream has been abandoned. If it has then what is the purpose of this addon then?
